.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules
 package-lock.json
+yarn.lock
```

Comment:
`yarn.lock` shouldn't be ignored since `package.lock` is used.

Reply:
I'm not sure, but i think @EmilTholin prefers to use npm on this project, and also, and also, to avoid contributors to accidentally commit their `yarn.lock` alongside `package-lock.json`, which can cause some conflicts (warned by the package manager during install time)

On the other hand, we could also stick with @mefechoel suggestion on https://github.com/EmilTholin/svelte-routing/pull/117#discussion_r551244058

Reply:
Oh, I didn't see that one. Then it is fine :)